tools/NEC/nec_sd_generator.py

```diff
@@ -1140,15 +1176,24 @@ def max_coupling_card():
         int_antenna_phases = calculate_broadened_phase(args.frequency * 1e6, args.antenna_spacing,
                                                        args.int_antennas, num_sub_arrays=2)
         for m_ant in range(0, args.antennas):
-            antenna_magnitudes.append(1)
+            if m_ant in antennas_down or m_ant in receivers_down:
```

Comment:
broadened beam not implemented yet so maybe you could just remove this change to this part of the if-else

Reply:
I will comment out for now. I don't want to remove it completely lest we forget to put it back in once broadened beam is implemented.